src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPrice = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const result = dailyPrice * days;
+
+  if (days >= 7) {
+    return days * dailyPrice - bigDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
And you don't need this additional check, cause you check the opposite of this condition in previous `if` clause
```suggestion
  if (days >= 3) {
```